### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by iamgiannisx** [snarfed/bridgy#1424] [General]
I just setup webmentions for my blog, but I have a question 
When I publish a post through my social network account (twiiter-mastodon) and someone respond to this post bridgy send the webmention to my site.
When somebody else post a link from my website at twitter or mastodon bridgy does not send them back as webmention, so my question is, this is a normal behavior ? or am I missing something.

Thanks.

**Comment by snarfed** [comment]
For Mastodon, sadly yes, this yes expected, since there's no global Mastodon search. https://brid.gy/about#which

For Twitter though, this should definitely work. Feel free to send me an example that didn't and I can look.

**Comment by iamgiannisx** [comment]
I just share a post at twitter from another account not mine and nothing came up to bridgy 

https://twitter.com/Social_Special/status/1630308186484875270

my account is https://twitter.com/iamgiannisx

**Comment by snarfed** [comment]
Thanks! Looks like this one is missing because Twitter's Search API, which is best effort only, isn't currently returning it. 😐 https://brid.gy/about#Twitter+responses

You can reproduce this with `twurl` or any other Twitter API client. Bridgy makes this call to search for links to your site:

```
twurl '/1.1/search/tweets.json?q=thecodebee.com&result_type=recent&count=50&include_entities=true&tweet_mode=extended&include_ext_alt_text=true'
```

Right now, the latest results it returns are on Feb 24. This may be expected, but it may also be the Twitter API itself gradually degrading. I trust it less and less these days. https://twittercommunity.com/t/the-twitter-api-is-now-effectively-unmaintained/186011

**Comment by iamgiannisx** [comment]
Thanks for your help.

I wait a few days and check again if something came up.
